modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-private-image-registry-private-azure"]
+= Optional: Preparing a private Microsoft Azure cluster for a private image registry
+
+By installing a private image registry on a private Microsoft Azure cluster, you can create private endpoints. Private endpoints disable public facing endpoints to the internal registry, adding an extra layer of security to your {product-title} deployment.
+
+Use the following guide to prepare your private Microsoft Azure cluster for installation with a private image registry.
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have prepared an `install-config.yaml` that includes the following information:
+** The `publish` field is set to `Internal`
+
+* You have set the permissions for creating an image registry storage account private endpoint. For more information, see "Azure permissions for installer-provisioned infrastructure".
```

Comment:
`image registry storage account private endpoint` is a mouth full. I think we can use just `private storage endpoint` or something in these lines. WDYT?
```suggestion
* You have set the permissions for creating a private storage endpoint. For more information, see "Azure permissions for installer-provisioned infrastructure".
```

Reply:
Fixed!